File: Basic/Conditions/219158I.go
package main

import (
	"fmt"
)

func solution() {

	var a, b int
	fmt.Scanf("%d %d", &a, &b)

	if a >= b {
		fmt.Printf("Yes\n")
	} else {
		fmt.Printf("No\n")
	}
}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}

File: Basic/Conditions/219158I.c
#include <stdio.h>
#define intll long long

void solution() {

	int a, b;
	scanf("%d %d", &a, &b);

	if (a >= b) {
		printf("Yes\n");
	} else {
		printf("No\n");
	}
}

int main() {
	// char TEST_CASE = !1;
	int t = 1;
	// if (TEST_CASE == 1) {
	// 	scanf("%d", &t);
	// }

	while (t--) {
		solution();
	}
	return 0;
}